JavadocParagraph: report violation with column

followup from https://github.com/checkstyle/checkstyle/pull/15710#issuecomment-2390504919

confusion was over:
```
$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="JavadocParagraph"/>
  </module>
</module>

$ cat Test.java
/**
 * Some Summary.
 *
 * <p><strong>1.2 <code>offset_table</code> structure</strong><p>
 */
public class Test {
}

$ java -jar checkstyle-10.18.1-all.jar -c config.xml Test.java 
Starting audit...
[ERROR] /var/tmp/Test.java:4: <p> tag should be placed immediately before the first word,
          with no space after. [JavadocParagraph]
Audit done.
Checkstyle ends with 1 errors.
```

attention that last P in line 4 is opening P. So violation on line 4 is valid but confusing on which P.

Expected: print column number for violation.

--- a/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java
+++ b/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java
@@ -49,10 +49,13 @@ class InputIncorrectJavadocParagraph {
    * <p><p>
    * <p> Some Javadoc.<p>
    */
-  // 2 violations 3 lines above:
+  // 3 violations 3 lines above:
+  //  '<p> tag should be preceded with an empty line.'
   //  '<p> tag should be placed immediately before the first word'
   //  '<p> tag should be preceded with an empty line.'
-  // 2 violations 5 lines above:
+  // 4 violations 6 lines above:
+  //  '<p> tag should be placed immediately before the first word'
+  //  '<p> tag should be preceded with an empty line.'
   //  '<p> tag should be placed immediately before the first word'
   //  '<p> tag should be preceded with an empty line.'
   class InnerInputCorrectJavaDocParagraphCheck {

--- a/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java
+++ b/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java
@@ -185,7 +185,7 @@ public class JavadocParagraphCheck extends AbstractJavadocCheck {
         final DetailNode nearestToken = getNearestNode(newline);
         if (nearestToken.getType() == JavadocTokenTypes.TEXT
                 && !CommonUtil.isBlank(nearestToken.getText())) {
-            log(newline.getLineNumber(), MSG_TAG_AFTER);
+            log(newline.getLineNumber(), newline.getColumnNumber(), MSG_TAG_AFTER);
         }
     }
 
@@ -197,22 +197,22 @@ public class JavadocParagraphCheck extends AbstractJavadocCheck {
     private void checkParagraphTag(DetailNode tag) {
         final DetailNode newLine = getNearestEmptyLine(tag);
         if (isFirstParagraph(tag)) {
-            log(tag.getLineNumber(), MSG_REDUNDANT_PARAGRAPH);
+            log(tag.getLineNumber(), tag.getColumnNumber(), MSG_REDUNDANT_PARAGRAPH);
         }
         else if (newLine == null || tag.getLineNumber() - newLine.getLineNumber() != 1) {
-            log(tag.getLineNumber(), MSG_LINE_BEFORE);
+            log(tag.getLineNumber(), tag.getColumnNumber(), MSG_LINE_BEFORE);
         }
 
         final String blockTagName = findFollowedBlockTagName(tag);
         if (blockTagName != null) {
-            log(tag.getLineNumber(), MSG_PRECEDED_BLOCK_TAG, blockTagName);
+            log(tag.getLineNumber(), tag.getColumnNumber(), MSG_PRECEDED_BLOCK_TAG, blockTagName);
         }
 
         if (!allowNewlineParagraph && isImmediatelyFollowedByNewLine(tag)) {
-            log(tag.getLineNumber(), MSG_MISPLACED_TAG);
+            log(tag.getLineNumber(), tag.getColumnNumber(), MSG_MISPLACED_TAG);
         }
         if (isImmediatelyFollowedByText(tag)) {
-            log(tag.getLineNumber(), MSG_MISPLACED_TAG);
+            log(tag.getLineNumber(), tag.getColumnNumber(), MSG_MISPLACED_TAG);
         }
     }
 

--- a/src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java
@@ -147,7 +147,7 @@ public class TreeWalkerTest extends AbstractModuleTestSupport {
     @Test
     public void testConditionRequiredWithoutOrdinaryChecks() throws Exception {
         final String[] expected = {
-            "7: " + getCheckMessage(JavadocParagraphCheck.class,
+            "7:5: " + getCheckMessage(JavadocParagraphCheck.class,
                     JavadocParagraphCheck.MSG_REDUNDANT_PARAGRAPH),
         };
         final String path = getPath("InputTreeWalkerJavadoc.java");

--- a/src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheckTest.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheckTest.java
@@ -59,29 +59,32 @@ public class JavadocParagraphCheckTest extends AbstractModuleTestSupport {
     @Test
     public void testIncorrect() throws Exception {
         final String[] expected = {
-            "14: " + getCheckMessage(MSG_LINE_BEFORE),
-            "15: " + getCheckMessage(MSG_LINE_BEFORE),
-            "22: " + getCheckMessage(MSG_LINE_BEFORE),
-            "24: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "34: " + getCheckMessage(MSG_LINE_BEFORE),
-            "36: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "47: " + getCheckMessage(MSG_LINE_BEFORE),
-            "47: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
-            "48: " + getCheckMessage(MSG_LINE_BEFORE),
-            "49: " + getCheckMessage(MSG_LINE_BEFORE),
-            "50: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "50: " + getCheckMessage(MSG_LINE_BEFORE),
-            "59: " + getCheckMessage(MSG_LINE_BEFORE),
-            "66: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
-            "72: " + getCheckMessage(MSG_LINE_BEFORE),
-            "84: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
-            "85: " + getCheckMessage(MSG_TAG_AFTER),
-            "96: " + getCheckMessage(MSG_LINE_BEFORE),
-            "98: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "101: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "101: " + getCheckMessage(MSG_LINE_BEFORE),
-            "111: " + getCheckMessage(MSG_TAG_AFTER),
-            "112: " + getCheckMessage(MSG_TAG_AFTER),
+            "14:4: " + getCheckMessage(MSG_LINE_BEFORE),
+            "15:19: " + getCheckMessage(MSG_LINE_BEFORE),
+            "22:21: " + getCheckMessage(MSG_LINE_BEFORE),
+            "24:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "34:13: " + getCheckMessage(MSG_LINE_BEFORE),
+            "36:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "47:8: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
+            "47:24: " + getCheckMessage(MSG_LINE_BEFORE),
+            "48:8: " + getCheckMessage(MSG_LINE_BEFORE),
+            "49:8: " + getCheckMessage(MSG_LINE_BEFORE),
+            "49:11: " + getCheckMessage(MSG_LINE_BEFORE),
+            "50:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "50:8: " + getCheckMessage(MSG_LINE_BEFORE),
+            "50:29: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "50:29: " + getCheckMessage(MSG_LINE_BEFORE),
+            "62:25: " + getCheckMessage(MSG_LINE_BEFORE),
+            "69:12: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
+            "75:29: " + getCheckMessage(MSG_LINE_BEFORE),
+            "86:12: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
+            "87:11: " + getCheckMessage(MSG_TAG_AFTER),
+            "97:27: " + getCheckMessage(MSG_LINE_BEFORE),
+            "99:13: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "102:36: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "102:36: " + getCheckMessage(MSG_LINE_BEFORE),
+            "112:11: " + getCheckMessage(MSG_TAG_AFTER),
+            "113:11: " + getCheckMessage(MSG_TAG_AFTER),
         };
         verifyWithInlineConfigParser(
                 getPath("InputJavadocParagraphIncorrect.java"), expected);
@@ -90,13 +93,13 @@ public class JavadocParagraphCheckTest extends AbstractModuleTestSupport {
     @Test
     public void testIncorrect2() throws Exception {
         final String[] expected = {
-            "14: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "h1"),
-            "22: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "ul"),
-            "24: " + getCheckMessage(MSG_LINE_BEFORE),
-            "38: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "table"),
-            "50: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "52: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "52: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "ol"),
+            "14:4: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "h1"),
+            "22:7: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "ul"),
+            "24:11: " + getCheckMessage(MSG_LINE_BEFORE),
+            "38:8: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "table"),
+            "50:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "52:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "52:8: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "ol"),
         };
         verifyWithInlineConfigParser(
                 getPath("InputJavadocParagraphIncorrect4.java"), expected);
@@ -105,34 +108,39 @@ public class JavadocParagraphCheckTest extends AbstractModuleTestSupport {
     @Test
     public void testAllowNewlineParagraph() throws Exception {
         final String[] expected = {
-            "16: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "16: " + getCheckMessage(MSG_LINE_BEFORE),
-            "17: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "17: " + getCheckMessage(MSG_LINE_BEFORE),
-            "28: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "28: " + getCheckMessage(MSG_LINE_BEFORE),
-            "30: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "39: " + getCheckMessage(MSG_LINE_BEFORE),
-            "41: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "56: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "56: " + getCheckMessage(MSG_LINE_BEFORE),
-            "56: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
-            "57: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "57: " + getCheckMessage(MSG_LINE_BEFORE),
-            "58: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "58: " + getCheckMessage(MSG_LINE_BEFORE),
-            "59: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "59: " + getCheckMessage(MSG_LINE_BEFORE),
-            "72: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "72: " + getCheckMessage(MSG_LINE_BEFORE),
-            "83: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "83: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
-            "86: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "88: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "89: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "89: " + getCheckMessage(MSG_LINE_BEFORE),
-            "100: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "100: " + getCheckMessage(MSG_LINE_BEFORE),
+            "16:4: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "16:4: " + getCheckMessage(MSG_LINE_BEFORE),
+            "17:19: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "17:19: " + getCheckMessage(MSG_LINE_BEFORE),
+            "28:21: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "28:21: " + getCheckMessage(MSG_LINE_BEFORE),
+            "30:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "39:13: " + getCheckMessage(MSG_LINE_BEFORE),
+            "41:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "56:8: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
+            "56:24: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "56:24: " + getCheckMessage(MSG_LINE_BEFORE),
+            "57:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "57:8: " + getCheckMessage(MSG_LINE_BEFORE),
+            "58:8: " + getCheckMessage(MSG_LINE_BEFORE),
+            "58:11: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "58:11: " + getCheckMessage(MSG_LINE_BEFORE),
+            "59:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "59:8: " + getCheckMessage(MSG_LINE_BEFORE),
+            "59:29: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "59:29: " + getCheckMessage(MSG_LINE_BEFORE),
+            "75:25: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "75:25: " + getCheckMessage(MSG_LINE_BEFORE),
+            "86:12: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "86:12: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
+            "89:12: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "91:12: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "92:29: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "92:29: " + getCheckMessage(MSG_LINE_BEFORE),
+            "103:12: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "103:12: " + getCheckMessage(MSG_LINE_BEFORE),
+            "103:31: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "103:31: " + getCheckMessage(MSG_LINE_BEFORE),
         };
         verifyWithInlineConfigParser(
                 getPath("InputJavadocParagraphIncorrect2.java"), expected);
@@ -141,15 +149,15 @@ public class JavadocParagraphCheckTest extends AbstractModuleTestSupport {
     @Test
     public void testAllowNewlineParagraph2() throws Exception {
         final String[] expected = {
-            "16: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
-            "17: " + getCheckMessage(MSG_TAG_AFTER),
-            "30: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "30: " + getCheckMessage(MSG_LINE_BEFORE),
-            "32: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "35: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "35: " + getCheckMessage(MSG_LINE_BEFORE),
-            "45: " + getCheckMessage(MSG_TAG_AFTER),
-            "46: " + getCheckMessage(MSG_TAG_AFTER),
+            "16:12: " + getCheckMessage(MSG_REDUNDANT_PARAGRAPH),
+            "17:11: " + getCheckMessage(MSG_TAG_AFTER),
+            "30:27: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "30:27: " + getCheckMessage(MSG_LINE_BEFORE),
+            "32:13: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "35:36: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "35:36: " + getCheckMessage(MSG_LINE_BEFORE),
+            "45:11: " + getCheckMessage(MSG_TAG_AFTER),
+            "46:11: " + getCheckMessage(MSG_TAG_AFTER),
         };
         verifyWithInlineConfigParser(
                 getPath("InputJavadocParagraphIncorrect3.java"), expected);
@@ -158,19 +166,19 @@ public class JavadocParagraphCheckTest extends AbstractModuleTestSupport {
     @Test
     public void testAllowNewlineParagraph3() throws Exception {
         final String[] expected = {
-            "15: " + getCheckMessage(MSG_LINE_BEFORE),
-            "17: " + getCheckMessage(MSG_LINE_BEFORE),
-            "20: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "20: " + getCheckMessage(MSG_LINE_BEFORE),
-            "34: " + getCheckMessage(MSG_LINE_BEFORE),
-            "38: " + getCheckMessage(MSG_LINE_BEFORE),
-            "44: " + getCheckMessage(MSG_LINE_BEFORE),
-            "48: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "52: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "52: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "h1"),
-            "67: " + getCheckMessage(MSG_LINE_BEFORE),
-            "80: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "80: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "ul"),
+            "15:12: " + getCheckMessage(MSG_LINE_BEFORE),
+            "17:15: " + getCheckMessage(MSG_LINE_BEFORE),
+            "20:9: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "20:9: " + getCheckMessage(MSG_LINE_BEFORE),
+            "34:11: " + getCheckMessage(MSG_LINE_BEFORE),
+            "38:8: " + getCheckMessage(MSG_LINE_BEFORE),
+            "44:8: " + getCheckMessage(MSG_LINE_BEFORE),
+            "48:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "52:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "52:8: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "h1"),
+            "67:11: " + getCheckMessage(MSG_LINE_BEFORE),
+            "80:8: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "80:8: " + getCheckMessage(MSG_PRECEDED_BLOCK_TAG, "ul"),
         };
         verifyWithInlineConfigParser(
                 getPath("InputJavadocParagraphIncorrect5.java"), expected);
@@ -179,8 +187,8 @@ public class JavadocParagraphCheckTest extends AbstractModuleTestSupport {
     @Test
     public void testJavadocParagraph() throws Exception {
         final String[] expected = {
-            "19: " + getCheckMessage(MSG_LINE_BEFORE),
-            "30: " + getCheckMessage(MSG_LINE_BEFORE),
+            "19:4: " + getCheckMessage(MSG_LINE_BEFORE),
+            "30:7: " + getCheckMessage(MSG_LINE_BEFORE),
         };
 
         verifyWithInlineConfigParser(

--- a/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect.java
+++ b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect.java
@@ -41,19 +41,22 @@ class InputJavadocParagraphIncorrect {
     boolean emulated() {return false;}
 
     // 2 violations 4 lines below:
-    //  'tag should be preceded with an empty line.'
     //  'Redundant <p> tag.'
+    //  'tag should be preceded with an empty line.'
     // violation 2 lines below 'tag should be preceded with an empty line.'
     /**<p>Some summary.<p>
      * <p>
      * <p><p>
      * <p>   Some paragraph.<p>*/
-    // violation 2 lines above 'tag should be preceded with an empty line.'
     // 2 violations 2 lines above:
+    //  'tag should be preceded with an empty line.'
+    //  'tag should be preceded with an empty line.'
+    // 4 violations 4 lines above:
+    //  '<p> tag should be placed immediately before the first word'
+    //  'tag should be preceded with an empty line.'
     //  '<p> tag should be placed immediately before the first word'
     //  'tag should be preceded with an empty line.'
      class InnerInputJavadocParagraphIncorrect {
-
         // violation 2 lines below 'tag should be preceded with an empty line.'
         /**
          * Some Summary.<p>
@@ -78,7 +81,6 @@ class InputJavadocParagraphIncorrect {
     }
 
     InnerInputJavadocParagraphIncorrect anon = new InnerInputJavadocParagraphIncorrect() {
-
         // violation 2 lines below 'Redundant <p> tag.'
             /**
          * <p>Some summary.
@@ -89,7 +91,6 @@ class InputJavadocParagraphIncorrect {
          */
         // violation 5 lines above 'Empty line should be followed by <p> tag on the next line.'
         public static final byte NUL = 0;
-
         // violation 3 lines below 'tag should be preceded with an empty line.'
         // violation 4 lines below 'tag should be placed immediately before the first word'
         /**

--- a/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect2.java
+++ b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect2.java
@@ -47,9 +47,9 @@ class InputJavadocParagraphIncorrect2 {
     boolean emulated() {return false;}
 
     // 3 violations 7 lines below:
+    //  'Redundant <p> tag.'
     //  '<p> tag should be placed immediately before the first word'
     //  'tag should be preceded with an empty line.'
-    //  'Redundant <p> tag.'
     // 2 violations 4 lines below:
     //  '<p> tag should be placed immediately before the first word'
     //  'tag should be preceded with an empty line.'
@@ -57,10 +57,13 @@ class InputJavadocParagraphIncorrect2 {
      * <p>
      * <p><p>
      * <p>   Some paragraph.<p>*/
-    // 2 violations 2 lines above:
+    // 3 violations 2 lines above:
+    //  '<p> tag should be preceded with an empty line.'
     //  '<p> tag should be placed immediately before the first word'
     //  '<p> tag should be preceded with an empty line.'
-    // 2 violations 4 lines above:
+    // 4 violations 5 lines above:
+    //  '<p> tag should be placed immediately before the first word'
+    //  '<p> tag should be preceded with an empty line.'
     //  '<p> tag should be placed immediately before the first word'
     //  '<p> tag should be preceded with an empty line.'
      class InnerInputJavadocParagraphIncorrect {
@@ -97,8 +100,10 @@ class InputJavadocParagraphIncorrect2 {
         boolean emulated() {return false;}
 
       /**
-       * * <p>/^WARN/   Some Summary.<p>*/
-      // 2 violations above:
+       * * <p>   Some Summary.<p>*/
+      // 4 violations above:
+      //  '<p> tag should be placed immediately before the first word'
+      //  '<p> tag should be preceded with an empty line.'
       //  '<p> tag should be placed immediately before the first word'
       //  '<p> tag should be preceded with an empty line.'
       int yyy = 99;

--- a/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheckExamplesTest.java
+++ b/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheckExamplesTest.java
@@ -35,8 +35,8 @@ public class JavadocParagraphCheckExamplesTest extends AbstractExamplesModuleTes
     @Test
     public void testExample1() throws Exception {
         final String[] expected = {
-            "16: " + getCheckMessage(MSG_LINE_BEFORE),
-            "21: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "16:4: " + getCheckMessage(MSG_LINE_BEFORE),
+            "21:4: " + getCheckMessage(MSG_MISPLACED_TAG),
         };
 
         verifyWithInlineConfigParser(getPath("Example1.java"), expected);
@@ -45,9 +45,9 @@ public class JavadocParagraphCheckExamplesTest extends AbstractExamplesModuleTes
     @Test
     public void testExample2() throws Exception {
         final String[] expected = {
-            "18: " + getCheckMessage(MSG_LINE_BEFORE),
-            "20: " + getCheckMessage(MSG_MISPLACED_TAG),
-            "23: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "18:4: " + getCheckMessage(MSG_LINE_BEFORE),
+            "20:4: " + getCheckMessage(MSG_MISPLACED_TAG),
+            "23:4: " + getCheckMessage(MSG_MISPLACED_TAG),
         };
 
         verifyWithInlineConfigParser(getPath("Example2.java"), expected);

